### ISSUE TITLE
Fahrenheit Temperatures

### ISSUE THREAD

**Issue by jsnyder** [Alexwijn/SAT#67]
I've been using another PID thermostat on HACS for a while for an electric heater I have on a porch and just ran across this one and wanted to try out the PID automatic tuning.  I set things up in the PID/PWM mode and then went to try a preset and found that on an HA setup that's set up to use Fahrenheit it drops in the Celsius presets and displays them unmodified with Fahrenheit units.  So, for example, if I select Comfort, It shows as if set to 20 F, which would be a bit chilly 😃 

<img width="490" alt="image" src="https://github.com/user-attachments/assets/9f5da605-7c54-47f6-9d0e-fdaf56b7d56d">

I'm not sure what the ideal solution would be here of where units should all just be converted to match the unit system selected in Settings -> System -> General or if some of the internals are dependent on Celsius values (beyond ranges/expectations for boiler temps). If I attempt to treat the preset values as if they were Fahrenheit, entering values that would be sensible in that unit are too high for the field validation. I'm going to leave it with the preset on none and the target temp set to 68 and see what happens.

Should the tuning work for an electric radiator (oil filled, very slow to warm up, no fan)?

**Comment by Alexwijn** [comment]
Correct, these are the default values. However, you can change them on the Presets screen in the configuration settings.

> Should the tuning work for an electric radiator (oil filled, very slow to warm up, no fan)?
It will require significant tweaking of the settings to keep the temperature stable. SAT is primarily designed for boilers.
